modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/exportimport/data/handler/BatchEnginePortletDataHandler.java

```diff
@@ -62,10 +61,7 @@ public BatchEnginePortletDataHandler(
 
 		_fileName = taskItemDelegateName + ".json";
 
-		setExportControls(
-			new PortletDataHandlerBoolean(
-				taskItemDelegateName, taskItemDelegateName, true, true, null,
-				className));
+		setEmptyControlsAllowed(true);
```

Comment:
Are you including the tests in this PR?

Reply:
As we talked privately, we are going to test just the instance level. Right now we have a pull that removes the validation (See [commit](https://github.com/liferay-headless/liferay-portal/pull/2627/commits/364aac5c85fb139df1d9d3b49d06303687f8fd06) in [pull](https://github.com/liferay-headless/liferay-portal/pull/2627)) just to have the tests green. One its merged I can add a commit to reflect the test logic here.